src/strategies/dynamic-whitelist/index.ts

```diff
@@ -0,0 +1,41 @@
+import fetch from 'cross-fetch';
+
+export const author = 'russellinho';
+export const version = '0.0.1';
+
+/**
+ * Gets all recipients of an NFT/token given from a smart contract
+ */
+const fetchVotingPower = async (
+  contractAddress: string,
+  etherscanApiKey: string
+): Promise<any> => {
+  const endpoint = 'https://api-rinkeby.etherscan.io/api?module=account&action=tokennfttx&contractaddress=' + contractAddress + '&apikey=' + etherscanApiKey;
```

Comment:
Also here, pass in snapshot block number to make sure results are not affected in future

```js
startblock=0&endblock=${blocktag}
```

Reply:
Maybe we should accept the host address from options and keep rinkeby as default?